artifactory.py

```diff
@@ -1887,10 +1887,12 @@ def creator(self):
         """
         return self._accessor.creator(self)
 
-    def is_dir(self):
+    def is_dir(self, follow_symlinks=False):
         """
         Whether this path is a directory.
         """
+        if follow_symlinks:
+            raise TypeError('Artifactory does not have symlink feature')
```

Comment:
Just ignore the flag IMO.

```suggestion
        if follow_symlinks:
            raise TypeError('Artifactory does not have symlink feature')
```

Reply:
@barneygale 

Thanks for your review.

The `is_dir` will also be exposed to real user. The `follow_symlinks` is follow the same way how Pathlib does, but it's just a placeholder, and not supported in Artifactory. It may be confused if the end user pass `follow_symlinks=True`.

The Zen of Python

> Errors should never pass silently.